Assets/Scripts/Screeps3D/RoomObjects/Views/CreepActionView.cs

```diff
@@ -160,7 +169,7 @@ public void Unload(RoomObject roomObject)
                     break;
                 case "attack":
                     _creep.ActionTarget = PosUtility.Convert(target, _creep.Room);
-                    EffectsUtility.Attack((_creep as IBump).BumpPosition);
+                    EffectsUtility.Attack((_creepRoot.transform.position + (Vector3)_creep.ActionTarget) / 2);
```

Comment:
What does this do? i'm not quite sure, could we get a comment or a meaningfull variable name for this position? :)

Reply:
BumpPosition is position towards which Bump action will happen (usually same as ActionTarget)
since we want to attack effect to happen not 'inside' Bump position (usually center of a cell so 'inside a creep/structure) but when the collision would happen(so more on the cell edge) we calculate midpoint position